templates/ministerial/ministerial-doc-en.html

```diff
@@ -0,0 +1,46 @@
+---
+title: Documentation about Ministerial profile page template
+description: Documentation and working examples for the Ministerial profile tempalte.
+language: en
+altLangPage: ministerial-fr.html
+dateModified: 2021-09-16
+---
+
+<div class="wb-prettify all-pre hide"></div>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+
+	<dt>Type</dt>
+	<dd>Canada.ca template</dd>
+
+	<dt>Last review</dt>
+	<dd>2021-09-15</dd>
+
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+
+	<dt>Specification</dt>
+	<dd><a href="https://design.canada.ca/mandatory-templates/ministerial-profile-pages.html">Ministerial profile pages - Canada.ca mandatory template</a></dd>
```

Comment:
At the moment, the right link to specifications page ends with a "-draft", like so:
```suggestion
	<dd><a href="https://design.canada.ca/mandatory-templates/ministerial-profile-pages-draft.html">Ministerial profile pages - Canada.ca mandatory template</a></dd>
```